File: exercicios/codigosEmOutrasLinguagens/EX6LI/EX6LI.py
def divisores(num):
    lista_divisores = []
    for i in range(1,num+1):    
        if num % i == 0:
            lista_divisores.append(i)
    return lista_divisores

def main():
    num = int(input("Digite um número para saber seus divisores: "))
    
    lista_divisores = divisores(num)
    print(f"Os divisores do número {num} são: {lista_divisores}")
    
if __name__ == "__main__":
    main()

File: exercicios/codigosEmOutrasLinguagens/EX6LI/EX6LI.c
#include <stdio.h>

int divisores(int num, int lista_divisores[]) {
    int contador = 0;
    for (int i = 1; i < num+1; i++) {
        if (num % i == 0) {
            lista_divisores[contador] = i;
            contador++;
        }
    }

    return contador;
}

int main() {
    int num;
    int lista_divisores[100]; 
    int quantidade;

    printf("Digite um número para saber seus divisores: ");
    scanf("%d", &num);

    quantidade = divisores(num, lista_divisores);
    printf("Os divisores do número %d são: ", num);
    for (int i = 0; i < quantidade; i++) {
        printf("%d ", lista_divisores[i]);
    }
    printf("\n");

    return 0;
}